src/main/com/github/soerxpso/portalfix/PortalFix.java

```diff
@@ -150,7 +154,7 @@ public void run() {
 						public void run() {
 							Player qp = Bukkit.getPlayer(this.ap);
 							if (qp == null) return; // else, they logged off.
-							qp.sendMessage(ChatColor.RED + Config.getPostMessage());
+							qp.sendMessage(doColors(Config.getPostMessage()));
```

Comment:
So I found a function over the weekend:

https://hub.spigotmc.org/javadocs/bukkit/org/bukkit/ChatColor.html#translateAlternateColorCodes(char,%20java.lang.String)

Sigh. We should probably use that :3 -- I'll be "updating" HiddenOre with it later on as well.